csrankings-x.csv

```diff
@@ -406,6 +406,7 @@ Xun Cao,Nanjing University,https://cite.nju.edu.cn/People/Faculty/20190621/i5054
 Xun Jian,Virginia Tech,http://people.cs.vt.edu/xunj/profile.html,m8ihzWwAAAAJ
 Xun Wang,University of Technology Sydney,https://www.uts.edu.au/staff/xun.wang-1,NOSCHOLARPAGE
 Xun Yi,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/y/yi-professor-xun,oDRazncAAAAJ
+Xunzhao Yin,Zhejiang University,https://scholar.google.com/citations?user=snOTdoIAAAAJ&hl=zh-CN,snOTdoIAAAAJ

```

Comment:
Google Scholar pages are not suitable home pages. Also, publication profile looks like EE/ECE, not CS. Please read the inclusion criteria. Please point to a departmental page and re-open, if appropriate.